### PR TITLE
Made tests tasks generators slightly more configurable from the outside

### PR DESCRIPTION
**Pull request by lorem--ipsum**

### CODE REVIEW

File: laborer.js [hunk 1]
```diff
@@ -206,43 +206,24 @@ exports.taskServerTypeScript = function(opt) {
...
-};
-
-exports.taskUtilsTest = function() {
+var generateTester = function(path, parameters) {
   return function() {
-    return gulp.src('./build/utils/**/*.mocha.js', {read: false})
+    return gulp.src(path, {read: false})
       // gulp-mocha needs filepaths so you can't have any plugins before it
-      .pipe($.mocha(mochaParams));
+      .pipe($.mocha(parameters));
```
Comment (vogievetsky) [inline laborer.js:213]: This is cool, I think it would be nice to default the parameters to `{ reporter: 'spec' }` if none are provided

Comment (vogievetsky) [inline laborer.js:213]: otherwise it is good to merge

Comment (lorem--ipsum) [inline laborer.js:213]: It's already the [default](https://mochajs.org/#reporters) :-)

**Comment by vogievetsky** [comment]
gr 🎱

**Pull request merged**